### PR TITLE
added fetching an image of the vehicle

### PR DESCRIPTION
**Pull request by m1n3rva**
Added feature to fetch the vehicle image :framed_picture:  from the server. To test it run `bimmerconnected image <user> <password> <vin>`. This will give you a new file `image.png`

I did not add any tests for this one as there is not much code anyways.

@gerard33 
This resolves your request #24 :heavy_check_mark: .

### CODE REVIEW

File: bimmerconnected [hunk 1]
```diff
@@ -34,6 +35,11 @@ def main() -> None:
     flash_parser.add_argument('vin', help='vehicle identification number')
     flash_parser.set_defaults(func=light_flash)
 
+    image_parser = subparsers.add_parser('image', description='flash the vehicle lights')
```
Comment (gerard33) [inline bimmerconnected]: description='save the vehicle image'

File: bimmerconnected [hunk 2]
```diff
@@ -78,6 +84,17 @@ def light_flash(args) -> None:
     print(status.state)
 
 
+def image(args) -> None:
+    """Trigger the vehicle to flash its lights."""
```
Comment (gerard33) [inline bimmerconnected]: """Save an image of the vehicle."""

File: bimmerconnected (hunk 1 above)
Comment (m1n3rva) [reply bimmerconnected]: thx. fixed it.

File: bimmerconnected (hunk 2 above)
Comment (m1n3rva) [reply bimmerconnected]: fixed

**Comment by gerard33** [comment]
@m1n3rva, when I run `bimmerconnected image <user> <password> <vin>` I get
```
usage: bimmerconnected [-h] {status,fingerprint,lightflash} ...
bimmerconnected: error: argument cmd: invalid choice: 'image' (choose from 'status', 'fingerprint', 'lightflash')

```

**Comment by lawtancool** [comment]
@gerard33 maybe you're not running latest version of the library? Since on your setup 'image' is not a valid choice, whereas on my clean install of the library, 'image' shows up as an option when I just run 'bimmerconnected' by itself.

**Comment by gerard33** [comment]
It’s the latest version. When i run `bimmerconnected` it shows the image option, using that however gives the above error message.

**Comment by gerard33** [comment]
@m1n3rva I have followed your instruction `bimmerconnected image <user> <password> <vin>` from your first post :wink:

When I run `./bimmerconnected image <user> <password> rest_of_world <vin>` I get
```
DEBUG:bimmer_connected.account:Getting vehicle list
DEBUG:bimmer_connected.account:getting new oauth token
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): b2vapi.bmwgroup.com
DEBUG:requests.packages.urllib3.connectionpool:https://b2vapi.bmwgroup.com:443 "POST /webapi/oauth/token HTTP/1.1" 200 None
DEBUG:bimmer_connected.account:got new token <token> with expiration date 2018-04-09 07:07:42.955858
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): b2vapi.bmwgroup.com
DEBUG:requests.packages.urllib3.connectionpool:https://b2vapi.bmwgroup.com:443 "GET /webapi/v1/user/vehicles HTTP/1.1" 200 None
Traceback (most recent call last):
  File "./bimmerconnected", line 109, in <module>
    main()
  File "./bimmerconnected", line 44, in main
    args.func(args)
  File "./bimmerconnected", line 96, in image
    image_data = vehicle.get_vehicle_image(400, 400, VehicleViewDirection.FRONT)
AttributeError: 'NoneType' object has no attribute 'get_vehicle_image'
```

**Comment by gerard33** [comment]
Yes, now it's working :+1:

**Pull request merged**